src/main/scala/uk/gov/nationalarchives/referencegenerator/Base25Encoder.scala

```diff
@@ -0,0 +1,17 @@
+package uk.gov.nationalarchives.referencegenerator
```

Comment:
At some point we should abstract this out to a separate library as we also use the same code in the API.